File: site-estatico/models/contato.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    let contato = sequelize.define('contato',{	
		fk_usuario: {
			field: 'fk_usuario',
			type: DataTypes.STRING,
			primaryKey: true,
			autoIncrement: true
		},	
        id_contato: {
			field: 'id_contato',
			type: DataTypes.INTEGER,
			primaryKey: true,
			autoIncrement: true
		},	
		valor: {
			field: 'valor',
			type: DataTypes.STRING,
			allowNull: true
		},
		fk_forma_contato: {
			field: 'fk_forma_contato',
			type: DataTypes.INTEGER, 
			allowNull: true
		},
	}, 
	{
		tableName: 'staff', 
		freezeTableName: true, 
		underscored: true,
		timestamps: false,
	});

    return contato;
};

File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/Contato.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class Contato {
    private Integer fk_usuario;
    private Integer id_contato;
    private String valor;
    private Integer fk_forma_contato;

    public Contato(Integer fk_usuario, Integer id_contato, String valor,
            Integer fk_forma_contato) {
        this.fk_usuario = fk_usuario;
        this.id_contato = id_contato;
        this.valor = valor;
        this.fk_forma_contato = fk_forma_contato;
    }
}
